pipeline/conf.py

```diff
@@ -97,7 +98,7 @@ def __getitem__(self, key):
         value = self.settings[key]
         if key.endswith(("_BINARY", "_ARGUMENTS")):
             if isinstance(value, string_types):
-                return tuple(shlex.split(value))
+                return tuple(shlex.split(value, posix=(os.sep == '/')))
```

Comment:
I had done this because I wasn't sure if any of the other possible values for `os.name` had posix-like paths. But it doesn't appear that they do, so that's functionally equivalent and definitely clearer.